policy/warn-k8s_ocp-deployment_deploymentconfig-bestpractices.rego

```diff
@@ -0,0 +1,450 @@
+package main
+
+warn[msg] {
+  apis := ["apps/v1", "apps.openshift.io/v1"]
+  kinds := ["Deployment", "DeploymentConfig"]
+
+  input.apiVersion == apis[_]
+  input.kind == kinds[_]
```

Comment:
These above lines are repeated loads, they could probs be abstracted into a function right?

Reply:
@springdo yep, we can reduce some duplication by creating utils rego files and importing them as 'data'

Reply:
I'll raise an issue for that, we can tackle it separately to this monster 🙈